### DISCUSSION TITLE
Main menu empty

### DISCUSSION THREAD

**Discussion by melvinmicallef** [Jyers/Marlin#817] [Q&A] [answered]
Hi, on commit 29885cd47e361b60b5647cccd025b632f996f750, when i boot the printer, the main menu only shows the text 'Main Menu' and stays blank. Printer is unresponsive when trying to connect with octoprint and rotating knob does nothing. I am using custom icons.

I have reverted back to 732cc15bf78e07a9067a08d3b61657f3d1449152 and it works ok.

![IMG_20210513_183827](https://user-images.githubusercontent.com/60214993/118157460-df186780-b41a-11eb-8c88-6bc1d118b0bf.jpg)

**Comment by tome9111991** [comment, marked as answer]
you read?
https://github.com/Jyers/Marlin/discussions/807

i have same 
but i don't know how far the version is at the moment.

**Reply by Tuncay-Ayhan** [reply]
Why don't you guys stick to the last known state? 

https://github.com/Jyers/Marlin/tree/8d59e8240ee6dd6b45efe31e4cdd9339229036d2

This is before the "move"

**Reply by melvinmicallef** [reply]
I am closing this discussion since i commented on the commit itself

**Comment by Jyers** [comment]
Weird, it wasn't doing that before. I'll have a look.

**Reply by tome9111991** [reply]
maybe should use original display firmware?
i use  a custom one

**Comment by Jyers** [comment]
Fixed now, it was being caused by #809 which I have now reverted. I'll have to investigate why exactly is was causing this issue.

**Reply by melvinmicallef** [reply]
Thank you :)
